Description: Informative message when using graph without graphviz
 Previously, trying to use the graph subcommand without graphviz being
 installed resulted in a cryptic error message: 
 .
 Can't exec "tred": No such file or directory at /usr/lib/perl/5.14/IO/File.pm line 66, <STDIN> line 1.
 .
 graphviz is already in the Suggests field of the debian package, and
 we cannot raise this dependency severity given the central role of
 quilt in the debian infrastructure. This informative error message is
 much more sensible.
Author: Martin Quinson <mquinson@debian.org>
Bug-Debian: http://bugs.debian.org/659944
Upstream-status: not submitted yet (half a dozen of patches are flying upstream)

Index: quilt.git/quilt/graph.in
===================================================================
--- quilt.git.orig/quilt/graph.in	2012-12-25 14:18:07.450999903 +0100
+++ quilt.git/quilt/graph.in	2012-12-27 15:12:29.302422537 +0100
@@ -17,6 +17,13 @@
 	. $QUILT_DIR/scripts/patchfns
 fi
 
+whichtred=`which /usr/bin/tred`
+if [ -z "$whichtred" ]
+then
+	echo $"It seems that graphviz is currently not installed (unable to find the tred binary). You must install it to use 'quilt graph'."
+	exit 1
+fi
+
 usage()
 {
 	printf $"Usage: quilt graph [--all] [--reduce] [--lines[=num]] [--edge-labels=files] [-T ps] [patch]\n"
